turbo.json

```diff
@@ -37,7 +37,9 @@
     },
     "//#manypkg": {},
     "//#knip": {},
-    "typesync": {},
+    "//#typesync": {
+      "inputs": ["**/package.json", ".typesyncrc"]
```

Comment:
uhm, either we don't specify the inputs or we add an empty `.typesyncrc` file. Otherwise the first time that we add a config file for typescync we are gonna run into trouble.

Reply:
I thought I'd added a `.typesyncrc` file, but even if not, is there an issue with specifying a non-existent file?

Reply:
> is there an issue with specifying a non-existent file?

Definitely yes. It creates confusion both now and for our future selfs.

Reply:
still waiting on this